pyOpenMS/manylinux/dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM quay.io/pypa/manylinux2014_x86_64
+
+RUN yum install -y wget
+RUN yum install -y xz qt5-qtbase-devel 
+# RUN yum install -y boost-regex boost-chrono boost-date-time boost-filesystem boost-iostreams boost-math
+# RUN yum install -y boost-python boost-serialization boost-system boost-thread 
+RUN yum install -y qt5-qtbase-devel qt5-qtsvg qt5-qtsvg-devel python-devel 
+RUN yum install -y libsvm-devel glpk-devel libzip-devel zlib-devel xerces-c-devel bzip2-devel sqlite-devel hdf5-devel
+RUN yum install -y libtool
+
+## this is hroest/manylinux2014_qt59:v1
+RUN yum install -y cmake3
+RUN ln -s /usr/bin/cmake3 /usr/bin/cmake
+
+## this is hroest/manylinux2014_qt59:v1.1
+
+# install contrib (need ALL here to build static libs)
+RUN git clone https://github.com/OpenMS/contrib.git && \
+    mkdir contrib-build && \
+    cd /contrib-build && \
+    cmake -DBUILD_TYPE=ALL ../contrib && \
+    rm -rf archives src && \
+    cd / && \
+    rm -rf contrib
+
+##     cmake -DBUILD_TYPE=SEQAN ../contrib && \
+##     cmake -DBUILD_TYPE=COINOR ../contrib && \
+##     cmake -DBUILD_TYPE=EIGEN ../contrib && \
+##     cmake -DBUILD_TYPE=WILDMAGIC ../contrib && \
+## 
+
+WORKDIR /
+
+## this is hroest/manylinux2014_qt59_contrib:v1.1
+
+
+# Metadata
+LABEL base.image="manylinux2014_x86_64"
+LABEL version="2.0"
+LABEL software="OpenMS (dependencies for manylinux2010)"
+LABEL software.version="qt5-v2.5.1"
```

Comment:
I think qt5 is now the default and can be removed. Also, 2.5**.1** is probably not necessary. I think hroest just made an update last time.
Also version can be 1.0 since it will be a new container.